web/pdf_viewer.js

```diff
@@ -223,6 +223,10 @@ class PDFViewer {
 
   #mlManager = null;
 
+  #onPageRenderedCallback = null;
+
+  #switchAnnotationEditorModeTimeoutId = 0;
```

Comment:
Given how this is being checked and reset in the code below, the initial value must be consistent.
```suggestion
  #switchAnnotationEditorModeTimeoutId = null;
```